pipeline/sc_population_estimates.json

```diff
@@ -15,7 +15,7 @@
 				},
 				"userProperties": [],
 				"typeProperties": {
-					"notebookPath": "/Impact Measurement Dashboards/Curation/statscan/clean_population_estimates"
+					"notebookPath": "/Nov 29 - MVP/Curation/statscan/annual_population_estimates"
```

Comment:
It seems like the DTS-STN/AP-Databricks repo doesn't have this change incorporated in its master branch
Can you ask Shaun to update the repo so the changes are reflected there as well